Mathlib/Algebra/Order/Field/Defs.lean

```diff
@@ -75,3 +75,15 @@ lemma inv_mul_right_le (ha : 0 ≤ a) : a * b⁻¹ * b ≤ a := by
 /-- Equality holds when `b ≠ 0`. See `inv_mul_cancel_right`. -/
 lemma le_inv_mul_right (ha : a ≤ 0) : a ≤ a * b⁻¹ * b := by
   obtain rfl | hb := eq_or_ne b 0 <;> simp [*]
+
+/-- Equality holds when `c ≠ 0`. See `mul_div_mul_left`. -/
+lemma mul_div_mul_left_le (ha : 0 ≤ a) (hb : 0 ≤ b) : c * a / (c * b) ≤ a / b := by
+  obtain rfl | hc := eq_or_ne c 0
+  · simpa using div_nonneg ha hb
```

Comment:
This is also true with both a and b non-positive, right? Let's add those too, for consistency with `div_nonneg`.

Reply:
What about this?